src/exrmod/gisobuild_exr_engine.py

```diff
@@ -2128,6 +2149,18 @@ def do_compat_check(self, repo_path, input_rpms, iso_key, eRepo):
                 elif 'signature: NOKEY' in line :
                     logger.debug("Ignoring RPM signing ")
                     continue
+                # Hack to ignore epoch if its present
```

Comment:
It would be preferable not to have words like "hack" in code we are sharing with our customers :-)

Reply:
taken care the comment